scripts/test.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
Can you move this to `tests`  as well please?

Reply:
Sry, it's a redundant test, I will remove it.